optaplanner-examples/pom.xml

```diff
@@ -175,6 +175,45 @@
         </exclusion>
       </exclusions>
     </dependency>
+    <dependency>
+      <groupId>javax.json</groupId>
+      <artifactId>javax.json-api</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.glassfish</groupId>
+      <artifactId>javax.json</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.mock-server</groupId>
+      <artifactId>mockserver-netty</artifactId>
```

Comment:
I declared it under <dependencyManagement>

Reply:
`<scope>` need to go under `<dependency>` because:
- the dependencyManagement stuff will be moved to a parent pom, so other pom.xml can use that dependency too
- other poms might use it in a different scope. So to avoid all confusion, we have a convention to have scope in the child pom (and never in the dependency management section), so it's explicitly clear without lookup.

Reply:
Can we remove the mockserver-netty usage?